src/pydartdiags/obs_sequence/obs_sequence.py

```diff
@@ -1134,6 +1146,50 @@ def create_header(self, n):
             self.header.append(copie)
         self.header.append(f"first: 1 last: {n}")
 
+    def replace_qc2_r8s(self):
```

Comment:
It looks like replace_qc2_r8s only acts on the dataframe. 

self is not needed here.

Reply:
what's the goal with having r8s in the function name?
The data could be r4, r8. 
Why not r`eplace_qc2_nan`?